test/integration/relay/connection.test.js

```diff
@@ -126,9 +127,19 @@ if (helper.sequelize.dialect.name === 'postgres') {
             values: {
               ID: {value: [this.Task.primaryKeyAttribute, 'ASC']},
               LATEST: {value: ['createdAt', 'DESC']},
+              CUSTOM: {value: ['updatedAt', 'DESC']},
               NAME: {value: ['name', 'ASC']}
             }
           }),
+          before: (options) => {
+            if (options.order[0][0] === 'updatedAt') {
+              options.order = Sequelize.literal(`
+                CASE
+                  WHEN completed = true THEN "createdAt"
+                  ELSE "updatedAt" End ASC`);
+            }
+            return options;
+          },
```

Comment:
```
      let orderBy = args.orderBy;
      let orderAttribute = orderByAttribute(orderBy);
      let orderDirection = args.orderBy[0][1];```

specifically this part of the connections configuration prevents that case because its looking directly for an array value. I tried setting it up directly in the order configuration first and it failed.